cluster/addons/fluentd-elasticsearch/es-image/Dockerfile

```diff
@@ -28,9 +28,11 @@ RUN apt-get update && \
 RUN cd / && \
     curl -O https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.5.2.tar.gz && \
     tar xf elasticsearch-1.5.2.tar.gz && \
-    rm elasticsearch-1.5.2.tar.gz
+    rm elasticsearch-1.5.2.tar.gz && \
+    mkdir -p /elasticsearch-1.5.2/config/templates
```

Comment:
This should be a separate `RUN` command since it's semantically unrelated to installing elasticsearch